### DISCUSSION TITLE
How do i do LE connection using mobly

### DISCUSSION THREAD

**Discussion by Avinashjha03** [google/mobly#929] [Q&A]
Requirement is :  I should be able to find an LE device and pair-connect to it using LE

**Comment by xpconanfan** [comment]
sorry this is not a mobly question.
you should study the Android API yourself

**Reply by Avinashjha03** [reply]
Hi I want to add the Android API for Ble connection in MOBLY to handle bluetooth LE test, like GATT server clients APIs are there, i want to add BLe connection API also for automating Bluetooth Low energy use case. I have gone through BLE API but unable to get info for this one, Any information/link will be of great help
